### PR TITLE
Add note about new Selenium 3.0 server

### PR DESCRIPTION
**Pull request by bocharsky-bw**
Fix https://github.com/KnpLabs/KnpUniversity/issues/701

The release note is: https://github.com/SeleniumHQ/selenium/releases

ping @weaverryan

### CODE REVIEW

**Comment by weaverryan** [comment]
👍 Is there anything else important that has changed for 3.0?

**Comment by bocharsky-bw** [comment]
@weaverryan Actually, yes, we should use [geckodriver](https://github.com/mozilla/geckodriver) for the newest version. But I can't figure out yet how to configure Behat with it properly and make it works.

**Comment by weaverryan** [comment]
Hmm, I thought that might be the case. Don't you just need to have this (and Selenium) running in the background? It sounds similar to the chromedriver, which worked like this (Behat doesn't know that chromedriver is running - you just set the browser, it tells Selenium about the browser, and then it uses the chromedriver in the background)

**Comment by bocharsky-bw** [comment]
Ok, I improve the note a bit... Well, so far I stuck on this bug https://github.com/SeleniumHQ/selenium/issues/2285 for `When I press "search_submit"` step. And looks like it bug was opened on Jun and still isn't closed. So, unfortunately, Firefox with Selenium 3 server for Behat tests isn't functional as for me yet or I missed its setup on my local machine. But I double checked and it doesn't work.

UPD:  But Chrome works fine for me.

File: knpu/install.md [hunk 1]
```diff
@@ -137,6 +137,12 @@ For me that's
 java -jar ~/Downloads/selenium-server-standalone-2.45.0.jar
 ```
 
+> Firefox `47.0.0` and fewer is not supported at all since Selenium `3.0.0` - update Firefox
```
Comment (weaverryan) [inline knpu/install.md]: fewer -> lower

File: knpu/install.md [hunk 2]
```diff
@@ -137,6 +137,12 @@ For me that's
...
+> to the latest version in order to use the latest Selenium server. Also you need to install
+> the new [geckodriver][geckodriver], but keep in mind that it's based on the evolving W3C
+> WebDriver spec, and uses the wire protocol in that spec, which is liable to change without
```
Comment (weaverryan) [inline knpu/install.md]: You lost me with everything after "but keep in mind". What should I watch out for and why exactly? Keep in mind, our users basically just want firefox to open and click around - they don't really care how it works (unless it's still buggy). And if chrome is better, we could add a note about using it instead (and how, I can't remember if we talk about using a different browser or not.

File: knpu/install.md (hunk 1 above)
Comment (bocharsky-bw) [reply knpu/install.md]: thanks!

File: knpu/install.md (hunk 2 above)
Comment (bocharsky-bw) [reply knpu/install.md]: Yes, I see, I simplified it

**Comment by weaverryan** [comment]
I like it, thanks!

**Pull request merged**